### ISSUE TITLE
Cannot edit expand/show more block in container as reusable block

### ISSUE THREAD

**Issue by bertiebaggio** [gambitph/Stackable#725]
**Describe the bug**

Adding a (Stackable) container block that contains a (Stackable) expand/show more block which is then saved as a (Wordpress) reusable block results in an error, and the block is unusable.

**To Reproduce**
1. Add a Container block
2. Add an Expand/Show More block to the container
3. Save the resulting block as a reusable block
4. Click "edit" on reusable block
5. See error:

"This block has encountered an error and cannot be previewed" in Gutenberg visual editor.

In console:

```
TypeError: "t is null"
    value https://example.org/wp-content/plugins/stackable-ultimate-gutenberg-blocks/dist/editor_blocks.js?ver=2.6.1:1
    value https://example.org/wp-content/plugins/stackable-ultimate-gutenberg-blocks/dist/editor_blocks.js?ver=2.6.1:1
    React 12
react-dom.min.js:103:487
TypeError: "this.getActiveTabPanelContainer() is undefined"
    value https://example.org/wp-content/plugins/stackable-ultimate-gutenberg-blocks/dist/editor_blocks.js?ver=2.6.1:1
    value https://example.org/wp-content/plugins/stackable-ultimate-gutenberg-blocks/dist/editor_blocks.js?ver=2.6.1:1
    React 12
```

In addition, the block cannot be edited either visually or as HTML.

**Expected behavior**

Block should remain editable after saving as reusable.

**Screenshots**

![stackable-reusable-error](https://user-images.githubusercontent.com/7524620/86596465-27acfc00-bf92-11ea-8258-26acdcc10a06.gif)

**Desktop (please complete the following information):**

 - OS: Linux
 - Browser: Firefox
 - Browser Version: 77.0.1 (64-bit)

**Comment by bertiebaggio** [comment]
Is there any further information I could provide on this that would help to track down and resolve the issue?

**Comment by bfintal** [comment]
@bertiebaggio Can you try it with WordPress 5.5.1? I tried replicating it now and it doesn't produce the error.

If it still does, would you be able to paste here the contents of your console log?

**Comment by blankart** [comment]
Already resolved in WP 5.5

**Comment by bertiebaggio** [comment]
I missed **bfintal**'s comment, but can confirm it is indeed resolved now. 

Thanks all 👍
